ecommerce/extensions/payment/views/hyperpay.py

```diff
@@ -42,7 +86,16 @@ class HyperPayResponseView(EdxOrderPlacementMixin, View):
     """
     Handle the response from HyperPay after processing the payment.
     """
-    SUCCESSFUL_TRANSACTION_RESULT_CODES_REGEX = re.compile(r'^(000\.000\.|000\.100\.1|000\.[36])')
+    SUCCESS_CODES_REGEX = re.compile(r'^(000\.000\.|000\.100\.1|000\.[36])')
+    SUCCESS_MANUAL_REVIEW_CODES_REGEX = re.compile(r'^(000\.400\.0[^3]|000\.400\.[0-1]{2}0)')
+    PENDING_CHANGEABLE_SOON_CODES_REGEX = re.compile(r'^(000\.200)')
+    PENDING_NOT_CHANGEABLE_SOON_CODES_REGEX = re.compile(r'^(800\.400\.5|100\.400\.500)')
+    PENDING_STATUS_URL_NAME = 'hyperpay:status-check'
+    PENDING_STATUS_PAGE_TITLE = _('HyperPay - Credit card - pending')
+    PENDING_STATUS_PAGE_MESSAGE = _(
+        'Your payment is being processed. This process can take up to 30 minutes. '
+        'Please stay on this page till the process is completed. Do not close or refresh the page.'
+    )
```

Comment:
Nit: It would be best if the `Do not close or refresh the page` was emphasized. No problem in keeping it this way for now though.

Reply:
@giovannicimolin, this is unused code as the string is embedded directly in the template [here](https://github.com/open-craft/ecommerce/pull/23/files#diff-5781de7e666b22e48062551351cf0136dbf0ee41a8bc71027032ef93f134c8a4R36-R39). Lines 36-39 in the `hyperpay_pending.html` template in case the link breaks after pushing a new commit.

I am removing it now.

Reply:
Removed in f0e0944b26079a3d2d02e5561aa2789529f31f09